web/styles/settings.css

```diff
@@ -513,11 +513,15 @@ input[type="checkbox"] {
     vertical-align: top;
     height: auto !important;
     width: auto !important;
+    white-space: break-spaces !important;
 
+    word-break: normal;
+    overflow-wrap: anywhere;
     background-color: transparent;
     border-radius: 4px;
     margin-top: 14px;
-    margin-left: 10px;
+    margin-bottom: 5px;
+    margin-left: 5px;
```

Comment:
These margin changes changes the UI for other notices too. I think we should only do CSS changes for the notice in that particular panel.

![Screenshot from 2023-02-27 22-55-09](https://user-images.githubusercontent.com/35494118/221636762-ae5ce355-af37-4e1f-b5e8-5ba7d9c416fd.png)

Reply:
have created a new classname `account-alert-notification` and made the new margin changes for the account alert notice only.